src/main/webapp/scripts/novncclient.js

```diff
@@ -0,0 +1,77 @@
+import RFB from 'https://cdn.jsdelivr.net/npm/@novnc/novnc@1.1.0/core/rfb.js';
+
+/**
+ * The NoVNCClient class essentially surrounds the RFB object of the noVNC
+ * library, taking care of remoting, what happens if someone connects, 
+ * and what happens if someone disconnects. The class also allows for 
+ * additional functionality that are specific to the scope of Virtual Movie
+ * Night, like changing the viewOnly property of the RFB object (or all
+ * owing who is able to controll).
+ */
+class NoVNCClient {
+  /**
+   * Initalizes a NoVNCClient object.
+   * @param {function(): void} connectCallback this function is called 
+   *    once the sessionScreen connects.
+   * @param {function(): void} disconnectCallback this function is called
+   *    once the sessionScreen disconnects.
+   * @param {HTMLElement} sessionScreenElement A block HTMLElement that 
+   *    specifies where the RFB object should attach itself.
+   */
+  constructor(connectCallback, disconnectCallback, sessionScreenElement) {
+    /**
+     * Represents the current noVNC RFB object of the 
+     * session, the single connection to the VNC server of the
+     * VM assigned to the session.
+     * @private {RFB}
+     */
+    this.sessionScreen_ = null; 
```

Comment:
Could you rename this? sessionScreen_ doesn't match what the object is. Rename it to rfb_connection_. Also, maybe explain briefly in the comment what RFB is